_pages/teaching.markdown

```diff
@@ -10,7 +10,49 @@ toc_sticky: true
 
 
 header:
-  image: /assets/images/wip.jpg
-  caption: "Photo credit: [**Gaelle Marcel**](https://unsplash.com/@gaellemarcel?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
+  image: /assets/images/teaching/teaching.jpg
+  caption: "Photo credit: [**Diego PH**](https://unsplash.com/@jdiegoph?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [**Unsplash**](http://unsplash.com/)"
 
 ---
+
+KBSS group main teaching scope is in Semantic Web, Ontologies and Enterprise architectures in Java. We also participate in teaching database systems.
+
+Ontologies, semantic web and enterprise architectures are also in the center of topics for theses in our group.
+
+## Subjects
+
+In the winter semester 2022/23 we teach following subjects:
+
+###  Ontologies and Semantic Web (b4m36osw)
+
+The course guides students through current trends and technologies in the semantic web field.
+
+[Subject description from the faculty web](https://fel.cvut.cz/en/education/bk/predmety/58/33/p5833706.html){: .btn .btn--info}
+[Details for students at CourseWare](https://cw.fel.cvut.cz/b221/courses/b4m36osw/start){: .btn .btn--info}
+
+### Enterprise Architecture (b6b36ear)
+
+The course offers an overview of enterprise system architectures, focusing on Spring and Java EE.
+
+[Subject description from the faculty web](https://fel.cvut.cz/en/education/bk/predmety/58/33/p5833906.html){: .btn .btn--info}
+[Details for students at CourseWare](https://cw.fel.cvut.cz/b211/courses/b6b36ear/start){: .btn .btn--info}
+
+
+In the summer semester, we participate in teaching of follosing subjects:
+
+### Software architectures (be4m36swa)
+
+In this course students become familiar with the general requirements for software architecture and related quality parameters that are monitored by software architectures.
+
+[Subject description from the faculty web](https://fel.cvut.cz/en/education/bk/predmety/48/79/p4879206){: .btn .btn--info}
+
+### Database systems (b0b36dbs)
+
+The course is designed as a basic database course mainly aimed at the student ability to design a relational data model and to use the SQL language for data definition as well as for data querying and to choose the appropriate degree of transaction isolation.
+
+[Subject description from the faculty web](https://fel.cvut.cz/en/education/bk/predmety/50/10/p5010606.html){: .btn .btn--info}
+[Details for students at CourseWare](https://cw.fel.cvut.cz/b211/courses/b0b36dbs/start){: .btn .btn--info}
+
+## Theses
+
+Topics for student theses are published in the [BPMPC system](https://bpmpc.feld.cvut.cz/dsfel/app/semesterprojecttopic/index.xhtml?all=true&department=14%2C&study=&field=&page=0&filter=), where it may be filtered by subjects, supervisors, department etc.
```

Comment:
where it may be filtered by title, supervisor, etc.

Reply:
removed department, not sure about title vs. subject

Reply:
done